wftests/warrior_tests/testcases/common_actions_tests/test_slurp_json.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
+<Testcase>
+        <Details>
+                <Name>test_slurp_json.xml</Name>
+                <Title>test_slurp_json</Title>
+                <Engineer>Sekhar</Engineer>
+                <Date>2018-11-30</Date>
+                <Time>12:08</Time>
+                <State/>
+                <InputDataFile>No_Data</InputDataFile>
+                <Datatype/>
+                <default_onError action="next"/>
+                <Logsdir/>
+                <Resultsdir/>
+                <ExpectedResults/>
+        </Details>
+        <Requirements>
+                <Requirement/>
+        </Requirements>
+        <Steps>
+                 <step Driver="common_driver" Keyword="store_in_repo" TS="1" draft="no">
+                        <Arguments>
+                                <argument name="filepath" value="slurp_content.json"/>
+                                <argument name="slurp" value="True"/>
+                        </Arguments>
+                        <Execute ExecType="Yes"/>
+                        <onError action="next"/>
+                        <Description>Using filepath(json file) option to slurp whole json content</Description>
+                        <iteration_type type="Standard"/>
+                        <context>positive</context>
+                        <impact>impact</impact>
+                        <runmode type="Standard" value=""/>
+                 </step>
+                 
+               <step Driver="common_driver" Keyword="verify_data" TS="1">
+                        <Arguments>
+                                <argument name="expected" value="30"/>
+                                <argument name="object_key" value="data_set1.var_str"/>
```

Comment:
Is the filepath value being stored in ``` data_set1.var_str ```? If so, then where is ``` data_set1.var_str ``` being initialized?